taichi/transforms/unreachable_code_elimination.cpp

```diff
@@ -73,15 +73,25 @@ class UnreachableCodeEliminator : public BasicStmtVisitor {
   }
 
   void visit(OffloadedStmt *stmt) override {
-    if (stmt->prologue)
-      stmt->prologue->accept(this);
+    if (stmt->tls_prologue)
+      stmt->tls_prologue->accept(this);
+
+    // TODO: should we use visit_loop here?
+    if (stmt->bls_prologue)
+      visit_loop(stmt->bls_prologue.get());
```

Comment:
`visit_loop` is to remove the last `ContinueStmt`. Is it possible that `bls_prologue` contains a `ContinueStmt`?

Reply:
Oh, I see. I don't think BLS blocks would contain any `ContinueStmt`. I'll update that to a simple `accept(this)`. Thanks for reviewing!